### PR TITLE
Ported Recorder classes and interface.

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #17 partly
**Changes**: Added the recorder classes and interface.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

### CODE REVIEW

**Comment by jddeep** [comment]
@mariobehling @CloudyPadmal please review the PR. Cheers!

File: app/src/main/java/io/neurolab/SerialRecorder.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    public void stop() {
+        if (out != null)
+            out.close();
+    }
+
+//    public static void main(String args[]) {
+//        (new Thread(new HelloRunnable())).start();
+//    }
```
Comment (CloudyPadmal) [inline app/src/main/java/io/neurolab/SerialRecorder.java]: Remove this code

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

**Comment by jddeep** [comment]
@CloudyPadmal Required changes have been seen into. You can now review the PR. Cheers!

**Comment by jddeep** [comment]
@CloudyPadmal Please take a look at the PR. Thanks.

**Pull request merged**